### DISCUSSION TITLE
Vulnerability Resolved ?

### DISCUSSION THREAD

**Discussion by abishek75** [spotbugs/spotbugs-maven-plugin#656] [General]
The High severity below is solved ?

https://spotbugs.github.io/spotbugs-maven-plugin/dependency-check-report.html

**Comment by hazendaz** [comment]
@abishek75 I transferred this to the correct location (maven plugin).  The CVE cannot be fixed nor an issue here.  It cannot be exploited.  Its part of maven.  Unless maven upgrades to something more recent there is nothing to do here.
